liferay-echo/liferay-portal
# LPS-194263 Adding the Convert to Page Template option to the ellipsis menu of the page

Motivation
=======
The motivation behind this ticket is to add the Convert to Page Template action in the Pages portlet so it is easier to access for the user.
https://liferay.atlassian.net/browse/LPS-194263

Proposed Solution
========
What I did to solve this problem is to re-use the CreateLayoutPageTemplateEntry and GetLayoutPageTemplateCollections action commands that are used in the Page Editor for the same action and adapt them to use it from the Pages portlet. I did the same with the ConvertToPageTemplate component.

Steps to Verify
========
1. Go to Site Builder -> Pages
2. Click on the ellipsis button for a page -> Click on Convert to Page Template action
3. In the case there aren't Page Templates sets, the modal should ask to create one an will create it and add the page template to it -> Go to Page Templates and check it is created fine
4. In the case there are Page Templates sets, should let you select one of them and add a page template to it
